### PR TITLE
Add typings for node pixel

### PR DESCRIPTION
**Pull request by ScreamZ**

### CODE REVIEW

File: types/node-pixel/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Type definitions for node-pixel 0.11
```
Comment (jakebailey) [inline types/node-pixel/index.d.ts:1]: You've created this package using the old method; you may want to retry using a later version of dts-gen or manually fix this up.

File: types/node-pixel/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "private": true,
+    "dependencies": {
+        "johnny-five": "^2.1.0"
```
Comment (jakebailey) [inline types/node-pixel/package.json:4]: You probably just need `@types/johnny-five` in here and nothing else. Check out one of the other packages in the repo for an example.

File: types/node-pixel/index.d.ts (hunk 1 above)
Comment (ScreamZ) [reply types/node-pixel/index.d.ts]: The new way looks really great! It's way easier to understand, I've updated things, thanks.

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## node-pixel ([<kbd>unpkg</kbd>](https://unpkg.com/browse/node-pixel@latest/))
was missing the following properties:
1. COLOR_ORDER

## Formatting

The following files are not formatted:
1. types/node-pixel/node-pixel-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 2f6ff4c694b1cb305796bb2f92275df3a09f90d9
</p>

**Comment by ScreamZ** [comment]
We should be good on that one :)

File: types/node-pixel/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    on(event: string, callback: () => void): void;
+    on(event: "error", callback: (e: Error) => void): void;
+}
+
+export class Pixel {
+    color(color: string | number[]): void;
+    colour(color: string | number[]): void;
+    off(): void;
+    clear(): void;
+}
```
Comment (peterblazejewicz) [inline types/node-pixel/index.d.ts:52]:
```suggestion
```
was this symbol exported in source?
https://github.com/ajfisher/node-pixel/blob/master/lib/index.js

Comment (ScreamZ) [reply types/node-pixel/index.d.ts:52]: No but could i do this using an exported interface? This way we will still be able to type the content of a pixel if we want it to be in a variable ?

```
export interface Pixel {
    color(color: string | number[]): void;
    colour(color: string | number[]): void;
    off(): void;
    clear(): void;
}
```

Comment (ScreamZ) [reply types/node-pixel/index.d.ts:52]: Also this is used in Strip.pixel `pixel(index: number): Pixel;`

Comment (ScreamZ) [reply types/node-pixel/index.d.ts]: @peterblazejewicz Used an exported interface so we can still type and this is correct.

**Comment by jakebailey** [comment]
The unrelated failure was just fixed on master, you should be able to merge to fix CI.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ScreamZ thanks!

**Comment by ScreamZ** [comment]
Ready to merge

**Pull request merged**